src/main.py

```
@@ -5,6 +5,7 @@
 from werkzeug.utils import secure_filename
 from PIL import Image
 from ultralytics import YOLO
+import logging
```

Pylint (4.0.4)

[convention] standard import "logging" should be placed before third party imports "flask.Flask", "werkzeug.utils.secure_filename", "PIL.Image", "ultralytics.YOLO"

(C0411)